### ISSUE TITLE
[UI-FIX]  Modify the look of dashboard balances

### ISSUE THREAD

**Issue by justfordaniel** [casinocoin/tokenwallet-mobile#96]
![Screensh](https://user-images.githubusercontent.com/57195813/73142852-55108800-408b-11ea-8ebd-67fe99c6d80b.png)
**Section or interface element**
i think the grey box with the csc balance could instead be deleted...... just grey the top half of the screen same grey as the bottom menu maybe or just the grey line? and then you have the whole top half to put a nice light background graphic with the balance centred.   i think that would look better but just personal preference.

also there is the small red dot below top left of the top menu bar.  not sure if thats reported already or not

**Reason**
Explain why this adjustment improves the look and feel 
or how it will align with branding guidelines

**Description**
A clear and concise description of how the look and feel should be.

**Screenshots**
Illustrate current look and feel. Or provide commented pictures, or 
edited screens to see how it should look like. Be clear about which picture is edited

**Comment by justfordaniel** [comment]
i see now ive sent gfg to it, it displays the gfg also....i think instead just either have a line like the history page... or grey backgroung - white background - grey background ect for tokens added.  the boxes stilll look odd to me.

if it started with centred csc balance.....and goes to top as moretokens added

**Comment by t3nshi** [comment]
I see. It is interesting the idea you suggest after you understand that there are not just a single balance displayed.  You mean that when you just have CSC to put that with no box on center, and when there is more token to put them on a list like in the history? 
by the way, what do you mean center... how it looks like for you?

**Comment by justfordaniel** [comment]
yes i mean never have an actual box surrounding the balance/logo 

i mean centred down a bit also when there is only csc.  

yes , as tokens are added a list like history/wallet lists which could be displayed in various other ways also.....  i am mainly against the box graphic though lol

**Comment by justfordaniel** [comment]
i assume it will have to scroll as many tokens are added also so there is potentially a lot of space to play with

**Comment by t3nshi** [comment]
The box in this case is called a card, and they are an UI element widespread on the app world that allows to convey information in an organized way.  It is not only an aesthetics thing by itself, is a bit more than that, given that in today app world it has "semantic" importance, which means it conveys not only a tidy and clean way to present information, but also has "meaning" by itself.  Even though we could consider getting rid of the card. I my opinion so far I would like to see a stronger argument to be in favor of that change though. Let's see if there are other people opinions that help us considering one way or another.

**Comment by t3nshi** [comment]
Although it is still using the card, we expanded the element to fill the width. So it is different on version 0.1.12 and 0.1.13 let's check this out and comment this accordingly.

**Comment by t3nshi** [comment]
Have you checked on the latest version yet?

**Comment by justfordaniel** [comment]
yeah, i had a look.   erm, its not what i was thinking tbh but then again ive thought of lots of random stuff lol.  i dont want to waste your time as im sure this part isn't your top priority but if you want rough ideas to consider i can do some to give you an idea of what im thinking if thats something you'd want?  i would need background images tho....if i do something worth exploring great and if not then at least there is less back and forwards texing.  up to you really, i dont know how else to continue this without mass confusion between us (or mainly me just confusing you) lol

**Comment by t3nshi** [comment]
don't worry we really appreciate your interest and involvement, thanks to the work you do on testing and reporting we have a better grasp of how things are perceived and what was missed and some point and needs improvement. What we have to do here is to put the ideas you have in a way they are understood, and bring the attention of more people, so they can assess if the changes work for them. When there is something that only one single person thinks that needs changing we will probably have it very low priority, But when there is something that everybody using the app is noticing and everyone things it should be this other way, we would know that it is something that we really do need now. 

I mean,for instance I want a lot of stuff on the app, but sometimes I discuss them with the team and they just dismiss them. Since it is a teamwork effort, I see that my need might not be something really average. Sometimes more people start asking for those and the team see the value of my interests and they gain some priority, sometimes they just don't.

**Comment by justfordaniel** [comment]
ok no worries.  i might throw some ideas your way if i get bored anyhow.  i totally get that its a sum of what everyone wants as opposed to what special requests ppl have....if i had it my way i would merge dashboard and wallet for example....but we all have totally different opinions that cant all be catered for

**Comment by t3nshi** [comment]
At some point there was someone asking to go back to the old boxes we are talking about here. Do you have any Idea so we get people involved here? If we are out of ideas I will close this issue in the mean time. Let me know.

**Comment by justfordaniel** [comment]
id agree the old boxes.  the new version is just wider as apposed to a different look

i dont know how to get people more involved here tbh sorry.  you could try ask deej to summon people in discord lol

yeah, probably best to close for now
thanks

**Comment by t3nshi** [comment]
When I said about an idea, I meant an actual idea for the layout change. The getting people involved it's just telling them on #preview discord that we need their opinion and posting the link to the issue.

**Comment by justfordaniel** [comment]
lmao sorry

ill try make some rough ideas quick
whats the font? i cant find it

**Comment by justfordaniel** [comment]
heres one way of doing it, just a rough idea... i couldn't find the font to fill it in but you get the idea anyhow.  i could think of other ways to do it also but this is kind of what i was thinking... along these lines anyhow

![image823](https://user-images.githubusercontent.com/57195813/74187937-6fc72d00-4c45-11ea-8357-277c59e5b668.png)

**Comment by justfordaniel** [comment]
you could fit send and recieve buttons in also so go straight from dashboard to send if you wanted

**Comment by justfordaniel** [comment]
another way it could work something like this coming from the sides

very rough as shapes not joined lol


![image8232](https://user-images.githubusercontent.com/57195813/74192454-79549300-4c4d-11ea-9fe1-8ba929c71fbd.png)

**Comment by justfordaniel** [comment]
let me know if you want more ideas of ways to do it

would need to know font tho because some of the other ideas i have would have no boxes or background img, just the text

**Comment by justfordaniel** [comment]
i got bored so i did another one... just used a random font
just another idea but more plain

![ancsc](https://user-images.githubusercontent.com/57195813/74273149-dca2fb80-4d07-11ea-930d-93fabf9ec640.png)

**Comment by justfordaniel** [comment]
the buttons are a bit shit but just an example of layout really 
my personal preference to a centre box with small writing is to use the space more basically... like any of the above examples...  hope this is useful anyhow...  stopping now lol

**Comment by t3nshi** [comment]
Thank you man. I already discussed a bit with the team and some of them proposed things similar and some did in a different direction. I will prototype base on that. And ultimately the changes may be included in an upcoming release. I'll make sure to let you know here when we add those changes. So far I cannot know how close will they be to what you suggest, I know some are great ideas and you got things moving anyway, but thanks for bringing many relevant points to our attention with your sketches. I know that we will end up with something better, and you have a hand in that for sure. 
Let's wait patiently for the new looks and then we may discuss them again.

**Comment by justfordaniel** [comment]
that's cool.  i certainly never expected it to look like what i suggested.   just wanted to help inspire some improvements.  i look forward to seeing it, thanks
